pkg/didcomm/crypto/jwe/authcrypt/encrypt.go

```diff
@@ -265,54 +249,25 @@ func (c *Crypter) buildRecipient(key string, apu, nonce []byte, tag string, send
 	return recipient, nil
 }
 
-// encryptOID will encrypt a msg (in the case of this package, it will be
-// the sender's public key) using the recipient's pubKey,
-// this is equivalent to libsodium's C function: crypto_box_seal()
-// https://libsodium.gitbook.io/doc/public-key_cryptography/sealed_boxes#usage
-// TODO add testing to ensure interoperability of encryptOID with libsodium's function above
-//      the main difference between libsodium and below implementation is libsodium hides
-//      the ephemeral key and the nonce creation from the caller while box.Seal requires these
-//      to be prebuilt and passed as arguments.
-// TODO replace 'OID' to 'SPK' recipient header which should represent the key in JWK encoded in a compact JWE format
-func encryptOID(recipientPubKey *[chacha.KeySize]byte, msg []byte) ([]byte, error) {
-	// generate ephemeral asymmetric keys
-	epk, esk, err := box.GenerateKey(randReader)
-	if err != nil {
-		return nil, err
-	}
-	// generate an equivalent nonce to libsodium's (see link in comment above)
-	nonce, err := generateLibsodiumNonce(epk[:], recipientPubKey[:])
+// encryptSymKey will encrypt symKey with the given kek and a newly generated nonce
+func (c *Crypter) encryptSymKey(kek, symKey []byte) (string, string, string, error) {
```

Comment:
The return values are unclear from the signature and code documentation.

Reply:
will update the code docs

Reply:
done